openpype/hosts/max/plugins/publish/validate_animation_timeline.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+
+from pymxs import runtime as rt
+from openpype.pipeline.publish import (
+    RepairAction,
+    ValidateContentsOrder,
+    PublishValidationError
+)
+from openpype.hosts.max.api.lib import get_frame_range, set_timeline
+
+
+class ValidateAnimationTimeline(pyblish.api.InstancePlugin):
```

Comment:
I dont think it is related to this PR  as it is from the max Host (also `extract_review_animation.py`  and `extract_thumbnailpy`)

Reply:
Thanks for notifying me, I tried to revert to a clean commit and I think I reverted back a lot. 
My bad!